Content.Server/Chemistry/EntitySystems/SolutionContainerSystem.cs

```diff
@@ -333,6 +335,27 @@ public FixedPoint2 GetReagentQuantity(EntityUid owner, string reagentId)
         return reagentQuantity;
     }
 
+    public bool TryGetMixableSolution(EntityUid uid,
+    [NotNullWhen(true)] out Solution? solution,
+    SolutionContainerManagerComponent? solutionsMgr = null)
+    {
+        if (!Resolve(uid, ref solutionsMgr, false))
+        {
+            solution = null;
+            return false;
+        }
+
+        var tryGetSolution = solutionsMgr.Solutions.FirstOrNull(x => x.Value.CanMix);
```

Comment:
Would be worth first firing an event in the target to try and let the target find the solution for us. (even if we don't use this right now it'll 100% be used in the future)

Reply:
did this via GetMixableSolutionAttemptEvent